easybuild/easyconfigs/o/OpenMM/OpenMM-7.4.1-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,66 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '7.4.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://openmm.org'
+description = "OpenMM is a toolkit for molecular simulation."
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/openmm/openmm/archive/']
+sources = ['%(version)s.tar.gz']
+patches = ['OpenMM-%(version)s_fix-Doxygen.patch']
+checksums = [
+    'e8102b68133e6dcf7fcf29bc76a11ea54f30af71d8a7705aec0aee957ebe3a6d',  # 7.4.1.tar.gz
+    '99d59e3106d9c9f9fbce3c3161117a37b2699e1dc09bc2a75619e0d880f29708',  # OpenMM-7.4.1_fix-Doxygen.patch
+]
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('Doxygen', '1.8.16'),
+]
+
+separate_build_dir = True
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('SWIG', '4.0.1'),
```

Comment:
need dep on GL, i.e. libglvnd here
platforms/opencl/src/cl.hpp includes GL/gl.h

Reply:
This applies to all `OpenMM` easyconfigs though, no?

Reply:
fixed in bb903f7

if fix is confirmed we can apply same fix to other OpenMM easyconfigs in separate follow-up PR